mozilla/blurts-server
# MNTOR-2681 - filter scan results on data broker ID, to avoid duplicate cards

# References: 
Jira: MNTOR-2681




# Description

# How to test

It looks like `monitoring` scans in some cases will have duplicate data broker IDs with new scan IDs.

This isn't possible to reproduce using the sandbox environment - I was able to reproduce the problem by inserting a `monitoring` scan in my local database:

- a new row in the `onerep_scans` table with the type `monitoring`
- new rows in `onerep_scan_results` that reference the above `onerep_scan_id`

Ensure that the new row(s) in `onerep_scan_results` are duplicating an existing `data_broker_id` - these are the only ones I've seen cause problems. Any genuinely new data brokers will be removed, but it looks like if it's already been removed, it stays in the "In progress" state indefinitely.

# Checklist (Definition of Done)
